bin/runtime/src/lib.rs

```diff
@@ -104,7 +106,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 46,
+    spec_version: 47,
```

Comment:
On Testnet we have now 51, please bump to at least 52. 

In the future, we'll use a smarter way to version runtimes, e.g. the one Polkadot uses, but for now let's stick with assumption that spec version on mainnet must be different than testnet, to avoid mistakes during deployments